test/python/asset/cdktf.json

```diff
@@ -0,0 +1,10 @@
+{
+  "language": "python",
+  "app": "pipenv run python main.py",
+  "terraformProviders": ["aws@~> 2.0"],
+  "terraformModules": ["terraform-aws-modules/vpc/aws@2.77.0"],
```

Comment:
Let's speed the test up and drop the aws provider / module in favour of the null provider.

Reply:
Good point 🙌

Reply:
Why use a provider at all? Or why not use a prebuilt one?

Reply:
Right :D there is no real need, will remove it

Reply:
Ok I won't, the tests call `cdktf get` which returns exit code 1 if no modules / providers are specified. I wouldn't want to 'fix' this inside the test harness since this is also annoying for CLI users. #690 captures that